DOC-1724: Update ToC and EditImage minimum plans (#2456)
tinymce/tinymce-docs: modules/ROOT/pages/editimage.adoc

:keywords: edit, image, rotate, rotateleft, rotateright, flip, flipv, fliph, editimage, imageoptions
:pluginname: Enhanced Image Editing
:plugincode: editimage
:pluginminimumplan: tiertwo

include::partial$misc/admon-premium-plugin.adoc[]

> The {pluginname} (`{plugincode}`) plugin adds a contextual editing toolbar to the images in the editor. If toolbar is not appearing on image click, it might be that you need to enable `+editimage_cors_hosts+` or `+editimage_proxy_service_url+` (see below).

Reviewer: ```suggestion
The {pluginname} (`{plugincode}`) plugin adds a contextual editing toolbar to images in the editor.

If the toolbar does not appear after clicking on an image, you may need to enable `+editimage_cors_hosts+` or `+editimage_proxy_service_url+` (see below).
```

Small Edits and some paragraphing (verbing weirds language).
